learning_journal_basic/tests/tests.py

```diff
@@ -10,10 +10,7 @@
 )
```

Comment:
Might be nice to have some tests that verify that CSRF functionality is working properly.